pkgs/development/python-modules/tt-flash/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  setuptools,
+  pyyaml,
+  tabulate,
+  pyluwen,
+  tt-tools-common,
+}:
+buildPythonPackage rec {
+  pname = "tt-flash";
+  version = "3.3.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tenstorrent";
+    repo = "tt-flash";
+    tag = "v${version}";
+    hash = "sha256-edWogH/HZZlGwyiqGbj6vunNxhsCr/+3LzmFgFGzjck=";
+  };
+
+  patches = [
+    # Remove when https://github.com/tenstorrent/tt-flash/pull/41 is merged
+    ./bump-pyyaml.patch
+  ];
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    tabulate
+    pyyaml
+    pyluwen
+    tt-tools-common
+  ];
+
+  meta = {
+    description = "Tenstorrent Firmware Update Utility";
+    homepage = "https://github.com/tenstorrent/tt-flash";
```

Comment:
```suggestion
    homepage = "https://tenstorrent.com";
    downloadPage = "https://github.com/tenstorrent/tt-flash";
```

Reply:
Homepage is set as this in the `pyproject.toml`, lets reflect that

Reply:
Done